client/extensions/wp-super-cache/state/data-layer/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import {
+  WP_SUPER_CACHE_TEST_CACHE,
+  WP_SUPER_CACHE_RECEIVE_TEST_CACHE_RESULTS,
+} from '../action-types';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { errorNotice, successNotice } from 'state/notices/actions';
+import { addHandlers } from 'state/data-layer/extensions-middleware';
+
+const testCache = ( { dispatch }, action ) => {
+	const { siteId, httpOnly } = action;
+	dispatch( http( {
+		method: 'POST',
+		apiVersion: '1.2',
+		path: `/jetpack-blogs/${ siteId }/rest-api/`,
+		query: {
+			path: '/wp-super-cache/v1/cache/test',
```

Comment:
this isn't as clear as with the WPCOM API layer but one thing that seems to work well there is organizing the file paths by the API path so that it's easy to work backwards from "what was calling `/wp-super-cache/v1/cache/test`?" to "it was the file in `/wp-super-cache/cache/test/index.js`